sonic_y_cable/y_cable.py

```diff
@@ -1237,3 +1262,90 @@ def get_nic_voltage_temp(physical_port):
         return -1
 
     return temp, voltage
+
+def get_internal_temperature(physical_port):
```

Comment:
Change from "internal" to "local" everywhere, as this is more descriptive that it is the temperature of the local connector, as opposed to the NIC or peer.

Reply:
fixed